### ISSUE TITLE
[Feature] Implement support for the LDAP served maps

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#905]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/901
- Created at 2011-06-18 01:16:00 by [dpal](https://pagure.io/user/dpal)
- Closed at 2020-03-24 14:22:30 as wontfix
- Assigned to nobody
---

https://bugzilla.redhat.com/show_bug.cgi?id=683518

SSSD should provide offline caching for the following maps:

In priority order:
    - High priority
        - services
        - protocols
    - Medium
        - aliases
        - bootparams
        - netmasks
        - ethers
        - networks
        - publickey
        - rpc
    - Low
        - hosts
        - printers

Before starting on each map please evaluate if implementing the map actually makes sense as the content of the map would be required to actually fetch the information (for example "hosts" map). 

Please split the ticket if needed.

### Comments

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2011-06-18 01:17:22](https://pagure.io/SSSD/sssd/issue/901#236783)**

Fields changed

description: SSSD should provide offline caching for the following maps:

In priority order:
 *High priority
 **services
 **protocols
 *Medium
 **aliases
 **bootparams
 **netmasks
 **ethers
 **networks
 **publickey
 **rpc
 *Low
 **hosts
 **printers

Before starting on each map please evaluate if implementing the map actually makes sense as the content of the map would be required to actually fetch the information (for example "hosts" map). 

Please split the ticket if needed. => SSSD should provide offline caching for the following maps:

In priority order:
 * High priority
 ** services
 ** protocols
 * Medium
 ** aliases
 ** bootparams
 ** netmasks
 ** ethers
 ** networks
 ** publickey
 ** rpc
 * Low
 ** hosts
 ** printers

Before starting on each map please evaluate if implementing the map actually makes sense as the content of the map would be required to actually fetch the information (for example "hosts" map). 

Please split the ticket if needed.

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2011-06-18 01:18:16](https://pagure.io/SSSD/sssd/issue/901#236784)**

Fields changed

description: SSSD should provide offline caching for the following maps:

In priority order:
 * High priority
 ** services
 ** protocols
 * Medium
 ** aliases
 ** bootparams
 ** netmasks
 ** ethers
 ** networks
 ** publickey
 ** rpc
 * Low
 ** hosts
 ** printers

Before starting on each map please evaluate if implementing the map actually makes sense as the content of the map would be required to actually fetch the information (for example "hosts" map). 

Please split the ticket if needed. => SSSD should provide offline caching for the following maps:

In priority order:
 * High priority
  * services
  * protocols
 * Medium
  * aliases
  * bootparams
  * netmasks
  * ethers
  * networks
  * publickey
  * rpc
 * Low
  * hosts
  * printers

Before starting on each map please evaluate if implementing the map actually makes sense as the content of the map would be required to actually fetch the information (for example "hosts" map). 

Please split the ticket if needed.

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2011-07-22 23:05:32](https://pagure.io/SSSD/sssd/issue/901#236785)**

Forking a ticket to track services map.

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2011-07-22 23:06:52](https://pagure.io/SSSD/sssd/issue/901#236786)**

New ticket is #929

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2011-07-22 23:08:41](https://pagure.io/SSSD/sssd/issue/901#236787)**

Fields changed

milestone: SSSD 1.8.0 => SSSD 1.9.0

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2011-12-10 19:57:11](https://pagure.io/SSSD/sssd/issue/901#236788)**

Fields changed

description: SSSD should provide offline caching for the following maps:

In priority order:
 * High priority
  * services
  * protocols
 * Medium
  * aliases
  * bootparams
  * netmasks
  * ethers
  * networks
  * publickey
  * rpc
 * Low
  * hosts
  * printers

Before starting on each map please evaluate if implementing the map actually makes sense as the content of the map would be required to actually fetch the information (for example "hosts" map). 

Please split the ticket if needed. => https://bugzilla.redhat.com/show_bug.cgi?id=683518

SSSD should provide offline caching for the following maps:

In priority order:
 * High priority
  * services
  * protocols
 * Medium
  * aliases
  * bootparams
  * netmasks
  * ethers
  * networks
  * publickey
  * rpc
 * Low
  * hosts
  * printers

Before starting on each map please evaluate if implementing the map actually makes sense as the content of the map would be required to actually fetch the information (for example "hosts" map). 

Please split the ticket if needed.
rhbz:  =>

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2011-12-16 16:00:36](https://pagure.io/SSSD/sssd/issue/901#236789)**

Fields changed

rhbz:  => [https://bugzilla.redhat.com/show_bug.cgi?id=683518 683518]

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2012-01-16 16:38:26](https://pagure.io/SSSD/sssd/issue/901#236790)**

Fields changed

blockedby:  => 
blocking:  => 
priority: major => blocker

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2012-02-02 16:13:20](https://pagure.io/SSSD/sssd/issue/901#236791)**

Fields changed

milestone: SSSD 1.9.0 => SSSD 1.9 beta
summary: Implement support for the LDAP served maps => [Feature] Implement support for the LDAP served maps

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2012-02-09 15:07:56](https://pagure.io/SSSD/sssd/issue/901#236792)**

Fields changed

feature_milestone:  => 
milestone: SSSD 1.9 beta => SSSD Deferred

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2013-01-03 16:16:42](https://pagure.io/SSSD/sssd/issue/901#236793)**

Fields changed

design:  => 
design_review:  => 0
fedora_test_page:  => 
priority: blocker => critical
selected:  =>

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2017-02-24 14:54:33](https://pagure.io/SSSD/sssd/issue/901#236794)**

**Metadata Update from @dpal**:
- Issue set to the milestone: SSSD Patches welcome

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2020-03-24 14:22:30](https://pagure.io/SSSD/sssd/issue/901#637037)**

Thank you for taking time to submit this request for SSSD. Unfortunately this issue was not given priority and the team lacks the capacity to work on it at this time.

Given that we are unable to fulfill this request I am closing the issue as wontfix.

If the issue still persist on recent SSSD you can request re-consideration of this decision by reopening this issue. Please provide additional technical details about its importance to you.

Thank you for understanding.

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2020-03-24 14:22:31](https://pagure.io/SSSD/sssd/issue/901#637038)**

**Metadata Update from @pbrezina**:
- Issue close_status updated to: wontfix
- Issue status updated to: Closed (was: Open)
